### ISSUE TITLE
Assertion failure

### ISSUE THREAD

**Issue by mglisse** [Ripser/ripser#44]
If I take the master branch and compile with `g++ -Wall ripser.cpp -o ripser -g` (note: no NDEBUG), running `./ripser --format point-cloud  examples/o3_1024.txt` produces
> ripser: ripser.cpp:115: index_t binomial_coeff_table::operator()(index_t, index_t) const: Assertion `n < B.size() && k < B[n].size() && n >= k - 1' failed.

Did I get the options wrong again? `./ripser --format lower-distance examples/projective_plane.lower_distance_matrix` produces the same error.

**Comment by mglisse** [comment]
It is just a swap in the assertion.
```diff
-       assert(n < B.size() && k < B[n].size() && n >= k - 1);
+       assert(k < B.size() && n < B[k].size() && n >= k - 1);
```

**Comment by mglisse** [comment]
Fixed in 01add51ff64aaf40889483260cc5c3b7d0f2a1e7.
